src/state/slices/opportunitiesSlice/resolvers/thorchainsavers/index.ts

```diff
@@ -155,6 +160,27 @@ export const thorchainSaversStakingOpportunitiesMetadataResolver = async ({
     }
   }
 
+  if (getConfig().REACT_APP_FEATURE_RUNEPOOL) {
+    stakingOpportunitiesById[thorchainAssetId as StakingId] = {
+      apy: undefined,
+      assetId: thorchainAssetId,
+      id: thorchainAssetId as StakingId,
+      provider: DefiProvider.ThorchainSavers,
+      // @TODO: calculate proper TVL
+      tvl: '10',
+      type: DefiType.Staking,
+      underlyingAssetId: thorchainAssetId,
+      underlyingAssetIds: [thorchainAssetId] as [AssetId],
+      rewardAssetIds: [thorchainAssetId] as [AssetId],
+      // Thorchain opportunities represent a single native asset being staked, so the ratio will always be 1
+      underlyingAssetRatiosBaseUnit: ['1'],
+      name: `RUNEPool`,
+      saversMaxSupplyFiat: '5000000',
```

Comment:
blocker: add a TODO or implement me

suggestion: either:
- use `MAXIMUMLIQUIDITYRUNE` mimir, since the caps for RUNEPool is determined by how much RUNE can go into the `runepool` module (i.e this limits limits the increase of pooled RUNE). Note the above is denominated in THOR base unit.
- hide caps for THORChain, making `saversMaxSupplyFiat` `string | undefined` and hiding the caps fill if `saversMaxSupplyFiat` is undefined

Prefer the latter for the sake of simplicity, as the max supply is really theoretical and at a low-level, but should never be hit with the current RUNEPool implementation.

Reply:
I did hide it by making `saversMaxSupplyFiat` undefined in [d387f46](https://github.com/shapeshift/web/pull/7430/commits/d387f46ef66e91fa6dc499cf57b03c1153695497)